basicMysql/register/change.php

```diff
@@ -0,0 +1,121 @@
+<?php
+	session_start();
```

Comment:
Too much space provided before session_start() command, could you please rectify it